### DISCUSSION TITLE
Black Square Around WeakAuras Icons

### DISCUSSION THREAD

**Discussion by Thokt** [SFX-WoW/Masque_Serenity#23] [Issues] [answered]
![f6icon](https://user-images.githubusercontent.com/34395403/33795319-fadcd2b6-dce7-11e7-8a6a-bacb076de1a3.jpg)

appeared after installing, can you help me get rid of them?

**Comment by StormFX** [comment]
Check your WeakAuras options and make sure you don't have something odd enabled.

**Comment by Thokt** [comment, marked as answer]
well, i disabled WeakAuras in the ElvUI addonskins and black squares disappeared

**Comment by StormFX** [comment]
Ah, yes. Forgot about ElvUI's AddonSkins module.
